by default, this load libpython$version.so
but that is in -dev - append the full version.
--
diff --git a/meson.build b/meson.build
index b0bc448..349cb93 100644
--- a/meson.build
+++ b/meson.build
@@ -43,7 +43,7 @@ message(f'python_abi_flags = @python_abi_flags@')
 pylib_loc = get_option('libpython-dir')
 fsmod = import('fs')
 pylib_prefix = 'lib'
-pylib_suffix = 'so'
+pylib_suffix = 'so.@SUFF@'
 pylib_ver = python_dep.version()
 pylib_locs = []
 if host_system == 'windows'
